src/components/page-header/navigation/navigation.component.ts

```diff
@@ -33,6 +35,11 @@ export class PageHeaderNavigationComponent implements AfterViewInit, OnDestroy {
 
     private readonly _onDestroy = new Subject();
 
+    get role(): string {
+        this._changeDetectorRef.detectChanges();
```

Comment:
We shouldn't run change detection inside a getter, if we think about what change detection does, it checks bindings in a template (and host bindings). 

In this case, the host binding calls the role getter to get the value and check if it has changed, which will trigger change detection, which checks the binding, which triggers change detection etc... and so on..

If we need to trigger change detection, it should be done in response to something, in this case it is in response to the menuItems changing, so we could so:

```ts
this.menuItems.changes.pipe(takeUntil(...)).subscribe(() => this._changeDetectorRef.detectChanges());
```